### PR TITLE
Refactor the initialization of examples

### PR DESCRIPTION
**Pull request by jordanjones243**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #140 

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

Refactor the initialization of the examples by exporting every example as a function and having a main parent function run all of the examples when the respective demo pages are loaded.

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Comment by jordanjones243** [comment]
Note for reviewer:

A `setTimeout` function is used in the `index.js` and`api.js` files for the examples due to a runtime issue where the calls to locate the html from the javascript files were receiving `null` values.

File: demo/index.html [hunk 1]
```diff
@@ -32,20 +32,23 @@
     <script type="module">
       import 'https://unpkg.com/marked@latest/marked.min.js';
```
Comment (jason-capsule42) [inline demo/index.html:33]: Lets switch all unpkg CDN use to the new jsdeliver CDN

**Review by blackfalcon — APPROVED** [review]
Next time, when you do more than one thing in a commit, break those out into different commits. The updates to a different CDN have nothing to do with updating how the examples work.

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**